recipes/rockhound/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "rockhound" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 22b6c90a5dc0e3c37aa2f899b850ddfd18b2c87c2b9d61632e6297870c11a4dc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pooch>=0.5
```

There needs to be a space here:
```suggestion
    - pooch >=0.5
```